TACC-Cloud/hazmapper
# task/wg-166-questionnaire-preview-title

## Overview: ##



## PR Status: ##


## Related Jira tickets: ##

* [WG-166](https://jira.tacc.utexas.edu/browse/WG-166)

## Summary of Changes: ##

- Use the activeFeature subscription display_path to pull the file name for asset detail panel.
- Added a function to extract file name from activeFeature display_path.
- Added a title property to the assets detail component 
- Added title to html asset detail template instead of the logic of picking out information from the features response
- Added padding to heading title since some questionnaire file names are very long


## Testing Steps: ##
1. Open up a project with a questionnaire 
2. Check that the title displays with the file name when either picking an asset from the asset browser or from clicking an asset from the map.

## UI Photos:
Other assets 
![Screenshot 2023-11-27 at 10 42 00 AM](https://github.com/TACC-Cloud/hazmapper/assets/96220951/e754f5ef-34bc-43ea-90fe-e0f5b41b22a2)


Questionnaire 
![Screenshot 2023-11-27 at 10 36 44 AM](https://github.com/TACC-Cloud/hazmapper/assets/96220951/0128da70-4283-4fa9-8c95-2b26fde68768)
